modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/ModuleProcess.cs

```diff
@@ -120,6 +121,16 @@ public class ModuleProcess : IProcessControl {
                         kPublisherModuleStart.WithLabels(
                             identity.DeviceId ?? "", identity.ModuleId ?? "").Inc();
                         await workerSupervisor.StartAsync().ConfigureAwait(false);
+
+                        // Reporting runtime state on restart.
+                        // Reporting will happen only in stadalone mode.
+                        var standaloneCliOptions = new StandaloneCliOptions(_config);
```

Comment:
>var standaloneCliOptions = new StandaloneCliOptions(_config); [](http://example.com/codeflow?start=24&length=61)

I think we already have an instance of this. Can't we reuse it somehow?

Reply:
Changed the code to persist `RunInStandaloneMode` so that I do not need to recreate `StandaloneCliOptions`. Previously it was created in `ConfigureContainer()` method and later registered with DI but only in the case when we were running in standalone mode. And I actually just need to know if we are running in standalone mode or not, so doing that now.